cg/store/crud/read.py

```diff
@@ -1729,19 +1729,29 @@ def get_all_pools_to_deliver(self) -> list[Pool]:
         return records.all()
 
     def get_orders(self, orders_request: OrdersRequest) -> tuple[list[Order], int]:
-        order_query: Query = apply_order_filters(
+        orders: Query = self.filter_orders(orders_request)
+        total_count: int = orders.count()
+        orders: list[Order] = self.sort_and_paginate_orders(
+            orders=orders, orders_request=orders_request
+        )
+        return orders, total_count
```

Comment:
```suggestion
        orders: list[Order] = self.sort_and_paginate_orders(
            orders=orders, orders_request=orders_request
        )
        return orders, orders.count()
```

Reply:
That won't work, the count needs to be done after the filtering but before the pagination

Reply:
Ah, right 👍